Composer/packages/lib/ui-shared/src/components/DisplayReadme/DisplayReadme.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
nit: I'd name this ```DisplayMarkdownDialog.tsx``` because:
1. it renders markdown and it's re-usable
2. it renders it in a dialog

Reply:
resolved